packages/amplify-category-api/src/index.ts

```diff
@@ -139,6 +147,22 @@ export const initEnv = async (context: $TSContext): Promise<void> => {
     return;
   }
 
+  // proceed if there are any existing imported Relational Data Sources
+  const apiResourceDir = getAPIResourceDir(resourceName);
+  const pathToSchemaFile = path.join(apiResourceDir, RDS_SCHEMA_FILE_NAME);
+  if(fs.existsSync(pathToSchemaFile)) {
+    // read and validate the RDS connection parameters
+    const secretsKey = await getSecretsKey();
+
+    const envInfo = {
+      isNewEnv: context.exeInfo?.isNewEnv,
+      sourceEnv: context.exeInfo?.sourceEnvName,
+      yesFlagSet: _.get(context, ['parameters', 'options', 'yes'], false),
+      envName: envName
+    }
+    await configureMultiEnvDBSecrets(context, secretsKey, resourceName, envInfo);
```

Comment:
This wouldn't get called since the RDS schema file doesn't exist?

Reply:
Don't be call this only on commands 'generate-schema' or 'update-secrets'? Both the commands are disabled now.

Reply:
We also call this during the env add to automatically copy over the secrets to née environment.

Reply:
The secrets wouldn't exist on the current environment. Do you still create copy over to the new environment with blank values?

Reply:
That's right. This wouldn't be triggered w/o a RDS schema